Formula/rclone.rb

```diff
@@ -16,7 +17,15 @@ class Rclone < Formula
   depends_on "go" => :build
 
   def install
-    system "go", "build", "-tags", "brew", *std_go_args
+    tags = []
+    on_macos do
+      tags.push("brew")
+    end
+    tags.length >= 1 ? tags.insert(0, "-tags"): nil
+    system "go", "build",
+      "-ldflags", "-s -X github.com/rclone/rclone/fs.Version=v#{version}",
+      *tags,
+      *std_go_args
```

Comment:
This makes it a little easier to understand, although I'm not sure if I got the `array << stuff` syntax right
```suggestion
    args = *std_go_args
    on_macos do
      args << ["-tags", "brew"]
    end
    system "go", "build",
      "-ldflags", "-s -X github.com/rclone/rclone/fs.Version=v#{version}",
      *args
```

Reply:
oh cool :) i used `args += ["-tags", "brew"]`. i hope thats alright too. im not a ruby dev so :)